README.md

```diff
@@ -183,6 +184,7 @@ before.
 -   `--start [filepath]` path of the initial json to look for
 -   `--summary` output a summary of the license usage',
 -   `--unknown` report guessed licenses as unknown licenses.
+-    `--clarificationsFile` A file that describe the license clarifications for each package, see clarificationExample.json, any field available to the customFormat option can be clarified. The clarifications file can also be used to specify a subregion of a package's license file (instead reading the entire file).
```

Comment:
As the headline states _All options in alphabetical order_, I'd prefer having this line in line 160 instead. Sorry for being picky.